SConstruct

```diff
@@ -271,6 +271,8 @@ opts.Add(BoolVariable("scu_build", "Use single compilation unit build", False))
 opts.Add("scu_limit", "Max includes per SCU file when using scu_build (determines RAM use)", "0")
 opts.Add(BoolVariable("engine_update_check", "Enable engine update checks in the Project Manager", True))
 opts.Add(BoolVariable("steamapi", "Enable minimal SteamAPI integration for usage time tracking (editor only)", False))
+opts.Add("cache_path", "Path to the directory to use for the SCons cache, if left empty cache is not used", "")
+opts.Add("cache_limit", "Limit, in MB, for the SCons cache (default 1 GB)", 1024)
```

Comment:
These names are just what I picked, can go with other names if these are too general or vague, originally added `scons_` as a prefix but I feel it's redundant for arguments to scons